Gtest/AudioClient.cpp

```diff
@@ -0,0 +1,160 @@
+/**
+ * Copyright (C) 2021-2022 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define LOG_TAG "AudioClient"
+#include "AudioClient.h"
+#include <log/log.h>
+
+using namespace std::chrono_literals;
+using namespace std;
+using namespace vhal::client::audio;
+using namespace vhal::client;
+
+int AudioClient::startDummyStreamerForInput()
+{
+    string       ip_addr("172.100.0.2");
```

Comment:
I dont think you can hardcode this IP, as this would be specific to instance0 I guess.
Can you verify this gtest on other instances too, paying attention to the logs of the corresponding instance.

Then, you can try to use the loopback IP("127.0.0.1") and see if it works per instance, and verify the same way.